packages/cli/generators/app/templates/README.md.ejs

```diff
@@ -0,0 +1,79 @@
+# <%= project.name %>
+
+This application is generated using [LoopBack 4 CLI](https://loopback.io/doc/en/lb4/Command-line-interface.html) with the
+[initial project layout](https://loopback.io/doc/en/lb4/Loopback-application-layout.html).
+
+## Install dependencies
+
+By default, dependencies have been installed when this application was generated.
+Whenever dependencies in `package.json` are changed, run the following command:
+
+```sh
+npm install
+```
+
+To only install resolved dependencies in `package-lock.json`:
+
+```sh
+npm ci
+```
+
+## Run the application
+
+```sh
+npm start
+```
+
+You can also run `node .` to skip the build step.
+
+Open http://127.0.0.1:3000 in your browser.
+
+## Rebuild the project
+
+To incrementally build the project:
+
+```
+npm run build
+```
+
+To force a full build by cleaning up cached artifacts:
+
+```
+npm run clean
+npm run build
+```
+
+## Fix code style and formatting issues
+
+If `eslint` and `prettier` are enabled for this project, you can use the
+following commands to check code style and formatting issues.
+
+```sh
+npm run lint
+```
+
+To automatically fix such issues:
+
+```sh
+npm run lint:fix
+```
+
+## Other useful commands
+
+- `npm run migrate`: Migrate database schemas for models
+- `npm run openapi-spec`: Generate OpenAPI spec into a file
+- `npm run docker:build`: Build a Docker image for this application
+- `npm run docker:run`: Run this application inside a Docker container
+
+## Tests
```

Comment:
is this npm test verbose or not?  I personally don't like seeing non-verbose '..........' when tests are running. CI is fine.
I like seeing verbose checkmarks etc. Do we want to also mention?
```
npm test -- -R spec
```

Reply:
No, that's too advanced for README.